tests/fuzz/runFuzzTest.sh

```diff
@@ -84,15 +86,21 @@ cp -r ./corpus/${TEST_NAME}/. "${TEMP_CORPUS_DIR}"
 if [[ ! -z "$AFL_FUZZ" && "$FUZZ_COVERAGE" != "true" ]]; then
     printf "Running %-s %-40s for %5d sec... " "${FIPS_TEST_MSG}" ${TEST_NAME} ${FUZZ_TIMEOUT_SEC}
     mkdir -p results/${TEST_NAME}
-    timeout ${FUZZ_TIMEOUT_SEC} afl-fuzz -i corpus/${TEST_NAME} -o results/${TEST_NAME} -m none ./${TEST_NAME}
+    set +e
+    timeout ${FUZZ_TIMEOUT_SEC} ${LIBFUZZER_INSTALL_DIR}/afl-fuzz -i corpus/${TEST_NAME} -o results/${TEST_NAME} -m none ./${TEST_NAME}
+    # See the timeout man page for specifics
+    if [[ "$?" -ne 124 ]]; then
```

Comment:
Could "$?" be overwritten by the time you try to echo it? I've seen (on the internet, ymmv) advice to stuff this in a var immediately after the command.

Reply:
Now that you mention it, I have an old scar somewhere from this issue...updated.